styles/octocat.css

```diff
@@ -0,0 +1,101 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+
+}
+
+ul {
+  list-style: none;
+  padding-left: 0;
+
+}
+
+img {
+  max-width: 100%;
+  border-radius: 2%;
+  border: 1px solid #ddd;
+}
+
+
+
+
+.section {
+
+
+
+}
+
+.container {
```

Comment:
You have the "container" here for the double container, just set the box model settings to make the content centered on larger screen widths.